kubernetes/samples/scripts/domain-lifecycle/startServer.sh

```diff
@@ -0,0 +1,143 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic managed server in a domain by patching
+  'spec.managedServers[<server-name>].serverStartPolicy' attribute of the domain
+  resource to 'ALWAYS'. It also increases the 'spec.clusters[<cluster-name>].replicas'
+  value for the managed server's cluster by '1'.  The 'spec.clusters[<cluster-name>].replicas'
+  value can be kept constant by using '-k' option.
+ 
+  Usage:
+ 
+    $(basename $0) -s myserver [-n mynamespace] [-d mydomainuid] [-k] {-m kubecli]
+  
+    -s <server_name>           : Server name parameter is required.
+
+    -d <domain_uid>            : Domain unique-id. Default is 'sample-domain1'.
+
+    -n <namespace>             : Domain namespace. Default is 'sample-domain1-ns'.
+
+    -k <keep_replica_constant> : Keep replica count constant. Default behavior is to increment replica count.
+
+    -m <kubernetes_cli>        : Kubernetes command line interface. Default is 'kubectl'.
+
+    -h                         : This help.
+   
+EOF
+exit $1
+}
+
+kubernetesCli=${KUBERNETES_CLI:-kubectl}
+serverName=""
```

Comment:
Minor point:  Among these var declarations, it looks like all but 'serverName' are used as local variables and are not directly accessed by the helper functions (I realize you can't mark them local since this is the global scope).  To make the code a little more readable, it'd be nice to treat serverName similarly - pass it in to the helper function instead of implicitly expecting it to be global...

Reply:
Fixed.